db-cleanup/airflow-db-cleanup.py

```diff
@@ -139,7 +141,7 @@ def cleanup_function(**context):
         session.commit()
         logging.info("Finished Performing Delete")
     else:
-        logging.warn("You're opted to skip deleting the db entries!!!")
+        logging.warning("You're opted to skip deleting the db entries!!!")
```

Comment:
typo? "You've"

Reply:
I've left the original messages written by teamclairvoyant. I could go down the route of fixing them all (and a few other areas) but my understanding is that we shouldn't divert from the original code too much.

This first step is to get this working in our unstable environment. Then we may consider how to proceed. Given the license used, we may end up creating our own at a later stage.

To be revisited once conceptually proven.